### DISCUSSION TITLE
where to get all the fits files for the testing phase?

### DISCUSSION THREAD

**Discussion by ishay320** [AstroPhotometry/AstroPhotometry#69] [Q&A] [answered]
where to get all the fits files for the testing phase?

**Comment by uriya222** [comment, marked as answer]
Search the website!!
see here : https://docs.astropy.org/en/stable/io/fits/index.html
and here: https://www.nhs.uk/live-well/healthy-weight/managing-your-weight/12-tips-to-help-you-lose-weight/
